### PR TITLE
Dynamic page (load teacher items using JavaScript)

### PR DESCRIPTION
**Pull request by dev89s**
1. Use Javascript to implement the teachers' list (speakers in the original design) dynamically from a Javascript Object
2. Styling adjustments for the course list on the desktop version
3. Styling the main top ribbon to add a hovering color change effect  
4. Add Readme to the project and format it professionally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+- An implementation of this portfolio is live on Github Pages(https://dev89s.github.io/webdev-inst/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎬 Representation <a id="representation"></a>
+
+- A representation of the project by Author himself. [Watch on loom](https://www.loom.com/share/bfaccfc8e55f49b8bc7c299d5e65aba6)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (IngDavidTM) [inline README.md]: - [x] While you did a great effort :+1: it is required for microverse to create a video with a maximum length of 5 minutes, which includes:
- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a id="acknowledgements"></a>
+
+I would like to thank all the coding partners that helped me through building this example.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
```
Comment (IngDavidTM) [inline README.md]: - [x] You did a wonderful job with creating the README file :+1:, but kindly don't forget to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). It should be done in the 'Acknowledgements' section

File: index.html [hunk 3]
```diff
@@ -115,84 +117,17 @@ <h3 class="course-heading">ASP.NET</h3>
...
       <a class="join-link" href="#">Join WebDev Academy 2023</a>
-      <a class="see-all-courses" href="">SEE ALL COURSES</a>
+      <div class="see-all-courses">
+        <a class="see-all-courses-link" href="">SEE ALL COURSES</a>
+      </div>
     </section>
 
     <!-- Program Teachers -->
     <section class="teachers">
       <h1 class="teachers-header">Program Teachers</h1>
```
Comment (IngDavidTM) [inline index.html]: - [x] You have done well in adhering to the correct hierarchy of `hx` elements :+1:. However, for semantic reasons, it is not recommended to use more than one `h1` element per page.

File: style.css
Comment (IngDavidTM) [inline style.css:1]: - [x] Great job with the styling of your project! :+1: However, as illustrated below, there are some issues with the height of certain elements when viewed on larger screens. Please make the necessary adjustments to fix these issues.
By fixing these height issues, you can improve the visual consistency of the website and ensure that all users, regardless of their screen size, can have an optimal viewing experience :+1:.


https://user-images.githubusercontent.com/105117832/225609497-591817b2-3673-40df-83a8-e316db42353a.mp4

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @dev89s  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] You have done an excellent job in following the semantics within your project :+1:. However, please keep in mind that elements like the five main courses, such as 'FrontEnd' and 'React,' should be enclosed within an `article` tag. This is because these elements are reusable and contain a significant amount of information.
By using the `article` tag, you will be indicating that these elements are standalone and can be reused on other pages within the website. It also enhances accessibility by allowing screen readers to understand the content more clearly, thereby improving the overall user experience.

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @dev89s, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**